core/modules/storyviews/classic.js

```diff
@@ -27,7 +27,7 @@ ClassicStoryView.prototype.navigateTo = function(historyInfo) {
 	var listItemWidget = this.listWidget.children[listElementIndex],
 		targetElement = listItemWidget.findFirstDomNode();
 	// Abandon if the list entry isn't a DOM element (it might be a text node)
-	if(targetElement === undefined || targetElement.nodeType === Node.TEXT_NODE) {
+	if(!targetElement || targetElement === undefined || targetElement.nodeType === Node.TEXT_NODE) {
```

Comment:
I think you can simplify this to:
`if(!targetElement || targetElement.nodeType === Node.TEXT_NODE) {`

Reply:
thanks, I'll do so @saqimtiaz